cw/cw305/util/spiflash.py

```diff
@@ -0,0 +1,27 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Utilities used to download firmware images into the OpenTitan system."""
```

Comment:
I don't think this should be a raw string.

Reply:
Done.